README.md

```diff
@@ -167,6 +167,55 @@ expect(getByTestId("val3").selected).toBe(true);
 The `values` parameter can be either an array of values or a singular scalar
 value.
 
+### `tab({shift, focusTrap})`
+
+Fires a tab event changing the document.activeElement in the same way the
+browser does.
+
+Options:
+
+- `shift` (default `false`) can be true or false to invert tab direction.
+- `focusTrap` (default `document`) a container element to restrict the tabbing
```

Comment:
What's the motivation here? Doesn't this create a [test user](https://kcd.im/test-user) scenario?

Reply:
@kentcdodds so let me start with the motivation for a focus trap option.  We are using focus-trap-react which keeps tabbing confined within an element.  Because jsdom doesn't have any way to deal with tabbing, these types of components won't work in jsdom.

So since jsdom doesn't seem to support tabs yet, focus trap basically won't work with userEvent.tab().  So does this introduce a test user?  Maybe.  But my hope is that jsDom can add support for tab and this entire feature can basically just adopt that.  In the meantime this seems like the least bad option.  That said if you have another way of writing a test that can let the user tab through a focus trap, I'd love to get some better ideas.

Here's the code for focus-trap which ftR uses under the hood.  
https://github.com/davidtheclark/focus-trap/blob/master/index.js

Reply:
That makes sense. I would add a note about the original intent for the option and why it should be used sparingly.

Then you've got a 👍 from me :) Thanks!

Reply:
Will do!  Thx Kent!